### PR TITLE
Try/code cleanup

### PR DESCRIPTION
**Pull request by stuwest**
Some code cleanup as we look ahead to putting into the .org plugin repo

### CODE REVIEW

File: class.wp-cldr.php [hunk 1]
```diff
@@ -146,6 +153,7 @@ public function get_cldr_locale( $wp_locale ) {
 		);
 
 		// Convert underscores to dashes and everything to lowercase.
+		$cleaned_up_wp_locale = '';
```
Comment (jblz) [inline class.wp-cldr.php]: This line will never affect the function output as the variable is reassigned in the next line without regard for this one.

File: class.wp-cldr.php [hunk 2]
```diff
@@ -309,7 +315,7 @@ public function flush_all_wp_caches() {
 	 * @return array An associative array where keys are WordPress locales and values are CLDR data items
 	 */
 	private function get_locale_bucket( $locale, $bucket ) {
-		if ( null === $locale ) {
+		if ( '' === $locale ) {
```
Comment (jblz) [inline class.wp-cldr.php]: Is the `null` check no longer desirable here at all? Is there any reason we couldn't just check `empty( $locale )` instead?

File: tests/wp-cldr-tests.php [hunk 3]
```diff
@@ -30,18 +30,31 @@ public function test_territory_name() {
 		// Test region names.
 		$this->assertEquals( 'Afrique', $this->cldr->territory_name( '002', 'fr_FR' ) );
 		$this->assertEquals( '亚洲', $this->cldr->territory_name( '142', 'zh-cn' ) );
+
+		// Test some bad slugs.
```
Comment (jblz) [inline tests/wp-cldr-tests.php]: Not sure I understand these test cases -- won't they fail?

File: class.wp-cldr.php (hunk 1 above)
Comment (stuwest) [inline class.wp-cldr.php]: yep good.  that was a few lines higher and I moved it down but isn't necessary now. will remove.

File: class.wp-cldr.php (hunk 2 above)
Comment (stuwest) [inline class.wp-cldr.php]: we used to pass `null` for an empty locale, but i got tripped up on a type juggling issue so decided to go with an empty string instead.  yes should be `empty()`. will change.

File: tests/wp-cldr-tests.php (hunk 3 above)
Comment (stuwest) [inline tests/wp-cldr-tests.php]: yep. the tests _should_ all just return a blank string.  tests are just checking to be sure we get that expected result.

this came up when some code coverage reports showed we weren't testing the handling of `null` / empty strings.

Comment (jblz) [inline tests/wp-cldr-tests.php]: Ok. I get it! Thanks.

**Comment by jblz** [comment]
:+1:

**Pull request merged**